ad_rss/src/situation/RssStructuredSceneNonIntersectionChecker.cpp

```diff
@@ -212,7 +214,8 @@ bool RssStructuredSceneNonIntersectionChecker::calculateLongitudinalRssStateSame
                                                         isSafe);
 
     data_non_intersection.longitudinal_relative_position = "Other in front";
```

Comment:
Is `to_string()` function can be used here?

Reply:
The same as previously -> original text is without ego/other word